EX/urls.py

```diff
@@ -14,8 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-
+from django.urls import path, include
+from EX.accounts.views import home as account_home
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('accounts/', include('allauth.urls')), # import account change urls
```

Comment:
This is not related to this PR. We can all Django all auto and set it up in other PR